### ISSUE TITLE
Crashes when sharing track

### ISSUE THREAD

**Issue by GoogleCodeExporter** [billycris007/open-gpstracker#345]
```
What steps will reproduce the problem?
1. Log a track
2. Click "Share track" (in German: "Track senden")

What is the expected output? What do you see instead?

"Unfortunately, Open GPS Tracker has stopped"

Which Android device are you using?

Galaxy Nexus (GSM)

What version of Android are you running?

4.1.1 Jelly Bean

What version of the Open GPS Tracker are you running?

1.3.2

Please provide any additional information below.

No special settings in this app. I don't know if it worked in earlier versions 
of Android on the Galaxy Nexus.
```

Original issue reported on code.google.com by `ischdem...@gmail.com` on 2 Aug 2012 at 10:38

**Comment by GoogleCodeExporter** [comment]
```
Same here (Motorola Defy, also on CM10/4.1 Jelly Bean) - was working before 
with CM7 Froyo. Unfortunately the log handling in 4.1 is different, so can't 
get error messages (yet).
```

Original comment by `iven....@gmail.com` on 5 Aug 2012 at 11:27

**Comment by GoogleCodeExporter** [comment]
```
I am also seeing the same problem with a uk galaxy nexus, 1.3.2-osmupdate 
android 4.1.1 JRO03C
Bit of a disabling bug as I don't seem to be able to get any data out of the 
app at all. Pitty as it was very good. Any idea when it will be fixed?
```

Original comment by `apkirkb...@gmail.com` on 7 Aug 2012 at 6:30

**Comment by GoogleCodeExporter** [comment]
```
Same here. Galaxy Nexus (GSM) - Android 4.1.1 Jelly Bean - Open GPS Tracker 
1.3.2-osmupdate. Data used by the application: 8.77 MB. Plenty of space left on 
the phone.
```

Original comment by `oliva.pr...@gmail.com` on 21 Aug 2012 at 8:16

**Comment by GoogleCodeExporter** [comment]
```
I also get a crash on 4.1.1.  The error is:

java.lang.SecurityException: Provider 
nl.sogeti.android.gpstracker/nl.sogeti.android.gpstracker.db.GPStrackingProvider
 does not allow granting of Uri permissions (uri 
content://nl.sogeti.android.gpstracker/tracks/3)
```

Original comment by `ginozemt...@gmail.com` on 26 Aug 2012 at 2:28

**Comment by GoogleCodeExporter** [comment]
```
Fixed in 108508f5ab7b 
```

Original comment by `rcgr...@gmail.com` on 6 Sep 2012 at 7:46
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
Issue 346 has been merged into this issue.
```

Original comment by `rcgr...@gmail.com` on 6 Sep 2012 at 7:46
